python/src/trezorlib/tools.py

```diff
@@ -105,7 +132,7 @@ def b58encode(v):
     return (__b58chars[0] * nPad) + result
 
 
-def b58decode(v, length=None):
+def b58decode(v: Union[str, bytes], expected_length: Optional[int] = None) -> bytes:
```

Comment:
why rename the argument?

Reply:
It just sounded more explicit to me (what length? - the length that is expected and it already tells us we will probably compare it to something). I am happy to revert it if it does not make sense

Reply:
i don't think `expected_length` is that much better, plus it's renaming an argument which might possibly break someone calling `b58decode(x, length=50)`.

so please revert.

Reply:
Reverted in 68dce6a